product_docs/docs/pgd/5/overview/index.mdx

```diff
@@ -51,7 +51,7 @@ Replicated data is sent in binary form when it's safe to do so.
 
 ### High availability
 
-Each master node can be protected by one or more standby nodes, so any node that goes down can be quickly replaced and continue. Each standby node can be either a logical or a physical standby node.
+Each master node can be protected by one or more standby nodes, so any node that goes down can be quickly replaced and continue. Each standby node is a logical standby node.
```

Comment:
Would this be the right place to also indicate we do not support physical standbys (plain Postgres feature) in combination with PGD? Or better add that to the limitations section?

Reply:
```suggestion
Each master node can be protected by one or more standby nodes, so any node that goes down can be quickly replaced and continue. Each standby node is a logical standby node. 
(Traditional Postgres physical standbys are not supported by PGD).
```

Reply:
Thanks, LGTM.